week-1/exercises/count.js

```diff
@@ -4,3 +4,4 @@ count = count + 1;
 
 // Line 1 is a variable declaration, creating the count variable with an initial value of 0
 // Describe what line 3 is doing, in particular focus on what = is doing
+console.log(count);
```

Comment:
You need to describe what line 3 is doing here

Reply:
Line 3 is an assignment statement that updates the value of the count variable.